.circleci/config.yml

```diff
@@ -19,6 +19,9 @@ jobs:
       - run:
           name: Install coveralls and gulp globally
           command: sudo yarn global add coveralls gulp
+      - run:
+          name: Install rsync
+          command: sudo apt install -y rsync
```

Comment:
I don't think we have another instance of using `apt` to install dependencies in the Circle CI config, but this should be OK. If it breaks, we'll fix it at the time.